packaging/docker/publish.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+# Cross-arch docker publish helper script
+# Needs docker in version >18.02 due to usage of manifests
+#
+# Copyright: SPDX-License-Identifier: GPL-3.0-or-later
+#
+# Author  : Pavlos Emm. Katsoulakis (paul@netdata.cloud)
+
+set -e
+
+WORKDIR="/tmp/docker" # Temporary folder, removed after script is done
+VERSION="$1"
+REPOSITORY="${REPOSITORY:-netdata}"
+MANIFEST_LIST="${REPOSITORY}:${VERSION}"
```

Comment:
Misleading variable name since this is not holding a list of anything and secondly docker image manifest is always a list. What you have here is a full image name.

Reply:
This variable, is the name of the manifest list that is created using the command "docker manifest create".
It is used for the manifest list name, that is why is called manifest list

Reply:
Manifest List is something like this: https://docs.docker.com/registry/spec/manifest-v2-2/#example-manifest-list

What you have here is merged image name and tag.

Reply:
Variable contains a name of a virtual image, naming it "manifest" is misleading.

Reply:
Yes, the variable name goes by it’s usage here, not by the content it has.
Having said that, this variable is used on the docker command to create the manifest list as per the documentation https://docs.docker.com/engine/reference/commandline/manifest/, thus the name MANIFEST_LIST because it contains the name we decide for the manifest list and consists of the virtual image name, practically.